source/mge-metadata.json

```diff
@@ -0,0 +1,2875 @@
+
```

Comment:
Rename file to `megengine-metadata.json`. Format using 2 leading space.

Reply:
done